public/app/plugins/panel/table2/module.tsx

```diff
@@ -35,5 +35,7 @@ export const plugin = new PanelPlugin<Options>(TablePanel)
       id: 'showHeader',
       name: 'Show header',
       description: "To display table's header or not to display",
+      defaultValue: true,
```

Comment:
```suggestion
```

Reply:
Should the property be "hide header"?  and default to empty?

Reply:
yea was considering this, the Table component property is named noHeader but for a display option not sure what is the most user-friendly / intuitive a default enabled option that you disable to have effect, or a default disable option that you enable to remove the heading. Will ask in #ux